src/main/java/application/MenuController.java

```diff
@@ -65,10 +70,38 @@ public SessionBean performInstall(@RequestBody InstallRequestBean installRequest
     @ApiOperation(value = "Make the given menu selection and return the next set of options, or a form to play.")
     @RequestMapping(value = Constants.URL_MENU_SELECT, method = RequestMethod.POST)
     public SessionBean selectMenu(@RequestBody MenuSelectBean menuSelectBean) throws Exception {
+        log.info("Select Menu with bean: " + menuSelectBean);
         MenuSession menuSession = getMenuSession(menuSelectBean.getSessionId());
         boolean redrawing = menuSession.handleInput(menuSelectBean.getSelection());
         menuRepo.save(menuSession.serialize());
-        return getNextMenu(menuSession, redrawing);
+        SessionBean nextMenu = getNextMenu(menuSession, redrawing);
+        log.info("Returning menu: " + nextMenu);
+        return nextMenu;
+    }
+
+    /**
+     * Make a a series of menu selections (as above, but can have multiple)
+     *
+     * @param menuSelectRepeater Give the selections to be made on the current MenuSession
+     *                             (could be a module, form, or case selection)
+     * @return A MenuResponseBean or a NewFormSessionResponse
+     * @throws Exception
+     */
+    @ApiOperation(value = "Make the given menu selection and return the next set of options, or a form to play.")
+    @RequestMapping(value = Constants.URL_MENU_SELECT_REPEATER, method = RequestMethod.POST)
+    public SessionBean selectMenuRepeater(@RequestBody MenuSelectRepeater menuSelectRepeater) throws Exception {
```

Comment:
Curious what you think the distinction should be - in my mind both these things would only be seen by us so didn't bother much to distinguish. Is it "things you'd always want to see vs. things you'd only be interested in when debugging" ?